hosting/templates/hosting/managevms.html

```diff
@@ -0,0 +1,52 @@
+{% extends "admin/base_site.html" %}
+{% block content %}
+
+<form action="{% url 'admin:createvm' %}" method="post">
+	{% csrf_token %}
+	<select id="vm_template" name="vm_template">
+		<option value="select">Select a template</option>
+		<option value="1">disk = 10GB, vcpu=1, ram=2GB</option>
```

Comment:
Also shouldn't we have VirtualMachineTemplate model? If we plan to cope with future requirements that is...

@telmich @levivm